src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerFullGroupTable.class.php

```diff
@@ -441,6 +433,37 @@ public function AddFields()
         $this->AddCustomColumns();
     }
 
+    /**
+     * @param array $fieldConfig
+     *
+     * @return null|array|string - returns the types AddColumn allows for the callback parameter
+     */
+    protected function getCellFormattingFunctionConfig(array $fieldConfig)
+    {
+        $formatFunctionName = $fieldConfig['callback_fnc'];
+
+        if ('' === $formatFunctionName) {
+            // handle UUID formatting globally if no callback method was defined
```

Comment:
Both (also below) could be moved to appropriately named methods to get rid of the comment.

Reply:
aded methods (in new PR with target 6.3.x)